### PR TITLE
Adding readiness probe for hello world samples

### PR DESCRIPTION
**Pull request by redborian**
In this PR, I am adding readiness probes for hello-world samples. Guestbook samples seem to have already implemented this using initContainers.

### CODE REVIEW

**Comment by redborian** [comment]
@simonz130 I have this arbitrary initialDelaySeconds in this PR. Is there a way to validate this? Verifying it in a cluster with a slow machine configuration is what comes to mind.

**Comment by sujit-kamireddy** [comment]
How about .net hello world app?

**Comment by redborian** [comment]
@simonz130 I have added readiness probes for dotnet sample as well. PTAL. Also, please advice on how to set initialDelay time over various samples.

File: dotnet/dotnet-hello-world/kubernetes-manifests/hello.deployment.yaml [hunk 1]
```diff
@@ -25,3 +25,7 @@ spec:
         env:
         - name: PORT
           value: "8080"
+        readinessProbe:
+          tcpSocket:
```
Comment (matthewmichihara) [inline dotnet/dotnet-hello-world/kubernetes-manifests/hello.deployment.yaml:29]: @redborian curious why we chose to use `tcpSocket` vs `httpGet`?

**Comment by matthewmichihara** [comment]
@redborian Re: `initialDelaySeconds`, my 2c is that we should set it along with `periodSeconds` to something small, like `2` since these containers do start up pretty quickly in my testing.

From what I understand, the guestbook samples use `initContainers` in the backend services to make sure the database is up and running before starting the backend container. It seems like we could still benefit from a `readinessProbe` on the frontend services in the guestbook samples since even with the `initContainers`, @russwolf ran into a situation where he tried to open a guestbook link from Skaffold and it wasn't ready yet: https://docs.google.com/document/d/1h2VKg3WZ9sHgNWXvmliq0UVJGUcdrbECSeYrjZYuVVw

**Comment by etanshaul** [comment]
@redborian should we get this merged in?

**Comment by etanshaul** [comment]
ping. @redborian @simonz130

**Pull request merged**